ui/prometheus-plugin/src/plugins/prometheus-time-series-query/get-time-series-data.ts

```diff
@@ -87,6 +87,7 @@ export const getTimeSeriesData: TimeSeriesQueryPlugin<PrometheusTimeSeriesQueryS
         name,
         values: values.map(parseValueTuple),
         formattedName,
+        labels: metric,
```

Comment:
Nice, yeah I probably should have done this back when I added `series_name_format`